Dockerfile

```diff
@@ -9,67 +9,71 @@ WORKDIR /rails
 
 # Set production environment
 ENV RAILS_ENV="production" \
-    BUNDLE_PATH="vendor/bundle" \
-    BUNDLE_WITHOUT="development:test"
+    BUNDLE_WITHOUT="development:test" \
+    BUNDLE_DEPLOYMENT="1"
 
-# Update gems and preinstall the desired version of bundler
-ARG BUNDLER_VERSION=2.3.24
+# Update gems and bundler
 RUN gem update --system --no-document && \
-    gem install -N bundler -v ${BUNDLER_VERSION}
+    gem install -N bundler
 
 
 # Throw-away build stage to reduce size of final image
 FROM base as build
 
 # Install packages needed to build gems and node modules
 RUN apt-get update -qq && \
-    apt-get install --no-install-recommends -y build-essential curl libpq-dev node-gyp pkg-config python-is-python3 unzip
+    apt-get install --no-install-recommends -y build-essential curl git libpq-dev node-gyp pkg-config python-is-python3
```

Comment:
Does including git here mean that the `/sha` action should work? I forget what the error was, but line 48 is `COPY --link . .` which I _guess_ copies dot-files directories? 🤔